packages/genesys-spark-tokens/snapshot-new/json/gse-flare-dark-semantic.json

```diff
@@ -24,7 +24,7 @@
           "30": "#252e78"
         },
         "tertiary": {
-          "10": "#000000",
+          "10": "rgba(0, 0, 0, 0)",
```

Comment:
This is going from black to transparent. Is that correct?

Reply:
Yes it was wrong before. ie the json files were missing their alpha channel info before this change.